content/en/docs/concepts/services-networking/dns-pod-service.md

```diff
@@ -16,13 +17,13 @@ Services with consistent DNS names instead of IP addresses.
 
 <!-- body -->
 
-Kubernetes DNS schedules a DNS Pod and Service on the cluster, and configures
-the kubelets to tell individual containers to use the DNS Service's IP to
-resolve DNS names.
+Kubernetes publishes information about Pods and Services which is used
+to program DNS.  Kubelet configures Pods' DNS so that running containers
+can lookup Services by name rather than IP.
 
-Every Service defined in the cluster (including the DNS server itself) is
-assigned a DNS name. By default, a client Pod's DNS search list includes the 
-Pod's own namespace and the cluster's default domain. 
+Services defined in the cluster are assigned a DNS name. By default, a
```

Comment:
possible nit (plural issue):

Each Service defined in the cluster is assigned a DNS name.

Reply:
You are right!